runtime/doc/develop.txt

```diff
@@ -233,11 +234,33 @@ definitions.  The |lua-vim| :help is generated from the docstrings.
 
 Docstring format:
 - Use LuaCATS annotations: https://luals.github.io/wiki/annotations/
-- Limited markdown is supported.
-  - List-items start with `-` (useful to nest or "indent")
+- Markdown is supported.
 - Use ``` for code samples.
   Code samples can be annotated as `vim` or `lua`
 - Use `@nodoc` to prevent documentation generation.
+- Use `@inlinedoc` to inline `@class` blocks into `@param` blocks.
+  E.g. >lua
+      --- Object with fields:
+      --- @class myopts
+      --- @inlinedoc
```

Comment:
Is there a difference if the `@inlinedoc` is before the class definition?

Reply:
Shouldn't be. The parser just looks for contiguous blocks of comments, and `@inlinedoc` just sets a flag in the object being built.

I'll add a test for this for good measure.